File: nomad-samples/admin/py/basic-live-channel-1.1/live_input/get_live_input_status_message.py
from live_input.get_live_input import *

def get_live_input_status_message(AUTH_TOKEN, INPUT_ID):
    # Get the live input
    INPUT = get_live_input(AUTH_TOKEN, INPUT_ID)

    # Check if input was found
    if (INPUT):
        # Check if there is status message
        if (INPUT["statusMessage"] and INPUT["statusMessage"]):
            # Return input status message
            return INPUT["statusMessage"]

    # There is no status message
    return ""

File: nomad-samples/admin/js/advanced-live-channel-1.1/live-input/get-live-input-status-message.js
import getLiveInput from "./get-live-input.js";

/**
 * Get Live Input Status Message
 *
 * @param {string} authToken    | Authorization token
 * @param {string} inputId      | The input ID for which to get the status message
 */
export default async function getLiveInputStatusMessage(authToken, inputId) {
    // Check for valid parameters
    if (!authToken || !inputId) {
        throw new Error("Get Live Input Status: Invalid API call");
    }

    // Get the live input
    const input = await getLiveInput(authToken, inputId);

    // Check if input was found
    if (input) {
        // Check if there is status message
        if (input.statusMessage && input.statusMessage) {
            // Return input status message
            return input.statusMessage;
        }
    }

    // There is no status message
    return "";
}
